<?xml version="1.0"?>
<launch>

  <rosparam command="load" file="$(find challenge_project)/config/turtlebot3_burger.yaml"/>
  <rosparam command="load" file="$(find challenge_project)/config/line_detection.yaml"/>

  <include file="$(find challenge_project)/launch/gazebo.launch">
  <arg name="x" value="0"/>
  <arg name="y" value="10.5"/>
  <arg name="z" value="0.0175"/>
  <arg name="roll" value="0"/>
  <arg name="pitch" value="0"/>
  <arg name="yaw" value="3.1415"/>
  <arg name="world" value="$(find challenge_project)/worlds/challenge1.world"/>
  </include>

  <node pkg="challenge_project" name="line_detection" type="line_following.py" required="true"/>

</launch>
